src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -2051,5 +1995,65 @@ string[] linkingTargetFields
                 RuntimeConfigValidator.ValidateEntityConfiguration(runtimeConfig);
             }
         }
+
+        /// <summary>
+        /// Validates that the runtime base-route is well-formatted and does not contain any reserved characeters and
+        /// can only be configured when authentication provider is Static Web Apps.
+        /// </summary>
+        /// <param name="runtimeBaseRoute">Value of runtime base-route as specified in config.</param>
+        /// <param name="authenticationProvider">The authentication provider configured.</param>
+        /// <param name="isExceptionExpected">Whether an exception is expected as a result of test run.</param>
+        /// <param name="expectedExceptionMessage">The expected exception message.</param>
+        [DataTestMethod]
+        [DataRow("/base-route", "StaticWebApps", false, DisplayName = "Runtime base-route correctly configured for Static Web Apps.")]
+        [DataRow(null, "StaticWebApps", false,
+            DisplayName = "Runtime base-route specified as null for Static Web Apps authentication provider passes config validation.")]
+        [DataRow(null, "AppService", false,
+            DisplayName = "Runtime base-route specified as null for Static Web Apps authentication provider passes config validation.")]
```

Comment:
nit: should AppService really pass?

Reply:
Yes, null base-route is ignored for every auth provider and hence it will pass for AppService.